node/src/chain_spec.rs

```diff
@@ -59,29 +67,70 @@ where
 /// Generate the session keys from individual elements.
 ///
 /// The input must be a tuple of individual keys (a single arg for now since we have just one key).
-pub fn pop_session_keys(keys: AuraId) -> pop_runtime::SessionKeys {
-	pop_runtime::SessionKeys { aura: keys }
+pub fn pop_devnet_session_keys(keys: AuraId) -> pop_runtime_devnet::SessionKeys {
+	pop_runtime_devnet::SessionKeys { aura: keys }
+}
+/// Generate the session keys from individual elements.
+///
+/// The input must be a tuple of individual keys (a single arg for now since we have just one key).
+pub fn pop_testnet_session_keys(keys: AuraId) -> pop_runtime_testnet::SessionKeys {
+	pop_runtime_testnet::SessionKeys { aura: keys }
 }
 
-pub fn development_config() -> ChainSpec {
-	// Give your base currency a unit name and decimal places
-	let mut properties = sc_chain_spec::Properties::new();
-	properties.insert("tokenSymbol".into(), "UNIT".into());
-	properties.insert("tokenDecimals".into(), 12.into());
+fn configure_for_relay(
+	relay: Relay,
+	properties: &mut sc_chain_spec::Properties,
+) -> (Extensions, u32) {
 	properties.insert("ss58Format".into(), 42.into());
+	let para_id;
 
-	ChainSpec::builder(
-		pop_runtime::WASM_BINARY.expect("WASM binary was not built, please build it!"),
-		Extensions {
-			relay_chain: "rococo-local".into(),
-			// You MUST set this to the correct network!
-			para_id: PARA_ID,
+	match relay {
+		// Test relay chains
+		Relay::ROC => {
+			para_id = 4385;
+			properties.insert("tokenSymbol".into(), "ROC".into());
+			properties.insert("tokenDecimals".into(), 12.into());
+
+			(Extensions { relay_chain: "rococo".into(), para_id }, para_id)
+		},
+		Relay::PAS => {
+			para_id = 4001;
+			properties.insert("tokenSymbol".into(), "PAS".into());
+			properties.insert("tokenDecimals".into(), 10.into());
```

Comment:
Why does Paseo have 10 decimals in stead of 12?

Reply:
Same as Polkadot

Reply:
Correct, it follows Polkadot runtime.